paper/supplementary_materials.tex

```diff
@@ -49,30 +52,30 @@
 \includegraphics[width=0.95\textwidth]{figs/decode_levels_kernels}
 \caption{\textbf{Across-participant timepoint decoding accuracy varies with
       correlation order and cognitive engagement across kernels.}
-    \textbf{a.-c.~Timepoint decoding accuracy as a function of order: PCA.} \textit{Order} ($x$-axis) refers to the maximum order of dynamic
+    \textbf{a.-c.~Timepoint decoding accuracy as a function of order: PCA.} Order ($x$-axis) refers to the maximum order of dynamic
     correlations that were available to the classifiers (see
-    \textit{Feature weighting and testing}).  The reported
```

Comment:
could you revert back the formatting in the supplemental materials?  i think it's ok to have italics and bold in the supplement

Reply:
added back italics formatting